java/integTest/src/test/java/glide/cluster/ClusterTransactionTests.java

```diff
@@ -73,6 +77,46 @@ public void info_simple_route_test() {
         assertTrue(((String) result[1]).contains("# Stats"));
     }
 
+    @SneakyThrows
+    @Test
+    public void test_xread_streams() {
+        String key9 = "{key}" + UUID.randomUUID();
+
+        ClusterTransaction transaction =
+                new ClusterTransaction()
```

Comment:
I think it is enough to test it in shared test

Reply:
I cannot compare the map result in the shared test :(

Reply:
why?

Reply:
because I get a unique stream id

Reply:
I think it will be something like
```java
Map.of("stream", Map.of("id", Map.of("key1", "value1", "key2", "value2"))))
```